events/models.py

```diff
@@ -3,17 +3,22 @@
 from django.contrib.auth.models import User
 from django.urls import reverse
 from django.core.exceptions import ValidationError
+from datetime import timedelta
 
 class Event(models.Model):
   def no_past(value):
     if value < timezone.now():
       raise ValidationError("The date cannot be in the past!")
-      
+
+  def more_than_48hrs(value):
+    if (timezone.now() + timedelta(hours=47,minutes=59) > value) and value > timezone.now():
```

Comment:
Can we set the cutoff time to 2-3 hrs before the event? We should be able to accommodate users who want to create day-of events. For these cases, user will not be able to make any changes once the event is created. Let me know what you think!